packages/nx/src/plugins/js/project-graph/build-dependencies/target-project-locator.ts

```diff
@@ -34,6 +34,11 @@ const defaultNpmResolutionCache: NpmResolutionCache = new Map();
 const builtInModuleSet = new Set<string>([
   ...builtinModules,
   ...builtinModules.map((x) => `node:${x}`),
+  // These are missing in the builtinModules list
+  // See: https://github.com/nodejs/node/issues/42785
+  // TODO(v20): We should be safe to use `isBuiltin` function instead of keep the set here (https://nodejs.org/api/module.html#moduleisbuiltinmodulename)
+  'test',
```

Comment:
Let's add a unit test for `test` and `node:test`.

Can you add `sqlite` and `sea` while you're at it?

Reply:
Sure, `node:sea` exists, `sea` does not so I'll add the former.

Reply:
Same with `node:sqlite`.